doc/NOTICE.rst

```diff
@@ -25,9 +25,9 @@ Read and understand the file ``LICENSE``; in particular, clause 7 (“Disclaimer
 
 Cite, at minimum, the following manuscript (exact citation TBD.):
```

Comment:
Since we have the zenodo entry as well, should we require that one to be cited as well?

Reply:
Okay I've added it as a "you may also cite"